src/react-native-animated-charts/src/helpers/ChartContext.ts

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Animated from 'react-native-reanimated';
+import * as redash from 'react-native-redash';
+
+export const enum CurveType {
+  basis = 'basis',
+  bump = 'bump',
+  linear = 'linear',
+  monotone = 'monotone',
+  natural = 'natural',
+  step = 'step',
+}
```

Comment:
we're not using `enum`s

```suggestion
export const CurveType = {
  basis: 'basis',
  bump: 'bump',
  linear: 'linear',
  monotone: 'monotone',
  natural: 'natural',
  step: 'step',
} as const;
```